### DISCUSSION TITLE
Error Delete Field Date

### DISCUSSION THREAD

**Discussion by hugopvl** [directus/directus#8639] [Archive]
Hey guys,

In rc.96 version when I try to delete a field, in this case of type Date, it gives the low error.

![image](https://user-images.githubusercontent.com/81326296/136533527-33b57b82-0955-44df-bc06-204558f73bdc.png)

**Comment by azrikahar** [comment]
This should be resolved once you do this (mentioned in the [rc.96 release note's notice](https://github.com/directus/directus/releases/tag/v9.0.0-rc.96)):

> - If you're upgrading from 95, and had some troubles with migrating due to "group" on directus_fields (#8369) on that version, please remove row `20210927A` from `directus_migrations` and re-run the migrations.

**Reply by hugopvl** [reply]
@azrikahar i remove the line from table directus_migrations  and then re-run npx directus database migrate:latest and then this error,

im doing something wrong ? 

![image](https://user-images.githubusercontent.com/81326296/136546482-4d67212a-0b7d-4862-a564-82006d0630ed.png)

**Reply by azrikahar** [reply]
hmm this was introduced in #8597, but unless you have any custom migrations, this error shouldn't really occur. Do you have any custom migrations?

**Reply by hugopvl** [reply]
no, i have a normal migration

**Reply by azrikahar** [reply]
that's odd. Would you mind double-checking do you have a `./extensions/migrations` folder in your Directus directory? 

@rijkvanzanten any idea what could be the cause here? Don't see there's anything wrong in that PR, but I might be missing something.. 🤔

**Reply by hugopvl** [reply]
As you can see, i don't have
![image](https://user-images.githubusercontent.com/81326296/136551200-d7ad1a9b-fdc4-440c-a7d2-9f0230dec260.png)

**Reply by hugopvl** [reply]
but the strangest thing is that in another project I tried to update, I didn't remove the line from the directus_migrations table and it gives me the same error, as well as the login error:

![image](https://user-images.githubusercontent.com/81326296/136551706-5c050b9e-820e-4dc6-9541-0c437aaca1a3.png)

![image](https://user-images.githubusercontent.com/81326296/136551796-47197911-a1aa-4492-89c6-557a65f6a61b.png)

**Reply by rijkvanzanten** [reply]
@hugopvl Mind sending me a database dump? Either here on or rijk@directus.io. I can take a look what's going on with these migrations and send you back a fixed copy 🙂 Those last couple query errors you showed are caused by missing migrations, which would make sense as your migrations wouldn't run due to that duplicate name issue.

**Reply by hugopvl** [reply]
They are already on their way, and thank you very much, to see if I can fix this.

**Comment by joselcvarela** [comment]
I've talked with @hugopvl and this issue resides between the monitor and the chair 😂
Basically, Hugo have edited manually a migration on `node_modules` and a backup file was created with `.save` suffix.
Migrations keys are based on file name without file extension so the conflict emerged.

Conclusion:
We were able to solve this by removing that `.save` temporary file and customizing some migrations (maybe we could use `drop[something]IfExists` on migrations, not sure if knex has support for all)

**Reply by rijkvanzanten** [reply]
Roger that!

**Reply by rijkvanzanten** [reply]
> (maybe we could use drop[something]IfExists on migrations, not sure if knex has support for all)

Yeah good question. On the one hand, we could use that to have migrations go through just fine, but at the same time, in cases like these, you'd actually want it to fail, so you have to stop and look into what went wrong. If the migration would've just silently worked, we'd never caught this particular issue 🤔

**Comment by hugopvl** [comment]
Really, the problem was really between the monitor and the chair :)
I didn't even notice the copy of the file, but thanks :D
